lib/noble.js

```diff
@@ -48,20 +48,35 @@ function Noble(bindings) {
     }
   }.bind(this));
 
+  //lazy init bindings on first new listener, should be on stateChange
   this.on('newListener', function(event) {
     if (event === 'stateChange' && !this.initialized) {
       this._bindings.init();
       this.initialized = true;
     }
   }.bind(this));
+
+  //or lazy init bindings if someone attempts to get state first
+  Object.defineProperties(this, {
```

Comment:
Overall this PR looks good to me.

Any idea what versions of Node.js added support for `Object.defineProperties`?